site/docs/.vuepress/config.ts

```diff
@@ -368,10 +359,6 @@ export default defineUserConfig({
                     text: "Heroku",
                     link: "/hosting/heroku.html",
                   },
-                  {
-                    text: "Google Cloud Functions",
-                    link: "/hosting/gcf.html",
-                  },
```

Comment:
Please revert this, as it clashes with #486

Reply:
Ok